src/pf/expand.lua

```diff
@@ -1236,6 +1236,10 @@ function selftest ()
       expand(parse("ether[0] = 2"), 'EN10MB'))
    assert_equals(expand(parse("src 1::ff11"), 'EN10MB'),
       expand(parse("src host 1::ff11"), 'EN10MB'))
+   assert_equals(expand(parse("proto sctp"), 'EN10MB'),
+      expand(parse("ip proto sctp or ip6 proto sctp"), 'EN10MB'))
```

Comment:
Ah, thanks, I should've put backslashes on those in the test.  (pflua already seems to accept either backslash or none to begin with on these)